lua/telescope/pickers.lua

```diff
@@ -369,7 +369,7 @@ function Picker:find()
     local process_result = function(entry)
       -- TODO: Should we even have valid?
       if entry.valid == false then
-        return
+        return -1
```

Comment:
I'm not sure why these have to return things (It's OK if they do, but I didn't read far enough to understand that)